src/sentry/api/invite_helper.py

```diff
@@ -17,50 +16,51 @@
 from sentry.utils import metrics
 from sentry.utils.audit import create_audit_entry
 
-INVITE_COOKIE = "pending-invite"
-COOKIE_MAX_AGE = 60 * 60 * 24 * 7  # 7 days
 
+def add_invite_details_to_session(request: Request, member_id: int, token: str):
+    """Add member ID and token to the request session
 
-def add_invite_cookie(request, response, member_id, token):
-    url = reverse("sentry-accept-invite", args=[member_id, token])
-    response.set_cookie(
-        INVITE_COOKIE,
-        urlencode({"memberId": member_id, "token": token, "url": url}),
-        max_age=COOKIE_MAX_AGE,
-    )
+    Args:
+        request: rest_framework.request.Request
+        member_id: int
+        token: str
+    """
+    request.session["invite_token"] = token
```

Comment:
@mdtro curious why we are doing it via session instead of a cookie. Is it security related?

Reply:
Yes, this is security-related. The cookie value can be manipulated to allow for invite code reuse.